.php_cs

```diff
@@ -9,6 +9,8 @@ return PhpCsFixer\Config::create()
         'concat_space' => ['spacing' => 'one'],
         'ordered_imports' => true,
         'random_api_migration' => true,
+        'phpdoc_var_without_name' => false,
+        'yoda_style' => false,
```

Comment:
Why are these options being set to `false`? We don't need PHPDoc blocks which list params and not their description and it's better to write conditions in [yoda style](https://en.wikipedia.org/wiki/Yoda_conditions) than not

Reply:
> We don't need PHPDoc blocks which list params 

Maybe you don't get it. `phpdoc_var_without_name = true` make
```php
/**
 * @var integer $i
 */
```

be wrong block. It make do `@var integer` without name

Reply:
> Yoda style

It this something that someone did on purpose?

Reply:
It's correct to report that example as wrong, because imho every variable, property or param should have a description. Just to be sure it's clear:

Wrong:
```php
/**
 * @var integer $i
 */
```

Correct:
```php
/**
 * @var integer $i ...
 */
```

Reply:
> it was done

And how you could merge something, that could not be passed by tests? 

https://github.com/getsentry/sentry-php/blob/dcc287f6579b765c7935e0e60fa642ae029b834f/lib/Raven/Serializer.php#L86
https://github.com/getsentry/sentry-php/blob/dcc287f6579b765c7935e0e60fa642ae029b834f/lib/Raven/ReprSerializer.php#L22 the whole block

and many many others

Reply:
>And how you could merge something, that could not be passed by tests?

I don't understand what you mean... Yoda conditions are just a way to write safer code by moving any constant to the left part of the condition. As an example, you could miss a `=` in an equal condition, and the value would then be assigned to the variable instead of compared to

```php
if ($var = true) {
    // $var is now true, instead of having been compared to
}
```

```php
if (true = $var) {
    // syntax error
}
```

Now, I understand that in an hypothetical world this would never happen, but it's better to prevent than to cure. To better understand the motivations behind yoda style btw, you can read the Wikipedia link I posted above

Reply:
Please look at this. This is CURRENT commit

https://github.com/getsentry/sentry-php/blob/dcc287f6579b765c7935e0e60fa642ae029b834f/lib/Raven/Serializer.php#L86
https://github.com/getsentry/sentry-php/blob/dcc287f6579b765c7935e0e60fa642ae029b834f/lib/Raven/ReprSerializer.php#L22

Reply:
@ste93cry , Nokitakaze is telling us that currently yoda conditions are not enabled, since the current codebase doesn't use them, and the code style checks are not failing.

@nokitakaze please try to explain your thoughts better, we are often having issues trying to understand your writing.

Reply:
Ok now I get the point. Sorry for misunderstanding your messages @nokitakaze but as @Jean85 said ofter I have issues understanding what you write

Reply:
I agree with not changing the current setting. We should otherwise investigate onto why that rule seems not to be enforced everywhere.

Reply:
There is the same problem in #505. https://travis-ci.org/getsentry/sentry-php/jobs/286575316

I thought PHP CS enabled these options, and so I disabled them

Reply:
Oh so IT IS currently failing. We should fix the CS then.